lib/crpc/middleware.go

```diff
@@ -85,6 +86,10 @@ func Validate(ls *gojsonschema.Schema) MiddlewareFunc {
 
 			result, err := ls.Validate(ld)
 			if err != nil {
+				if errors.Is(err, io.EOF) {
+					return cher.New(cher.BadRequest, nil, cher.New("body content isn't JSON", nil))
```

Comment:
I'd be inclined to use cher.M metadata with {"message": "invalid JSON"}

cher.New should never be called without a snake_case error code

Reply:
tbh cher.New should validate that 😂